src/photos/ducks/backup/index.jsx

```diff
@@ -10,16 +10,48 @@ import BackupInfo from './components/BackupInfo'
 import BackupActions from './components/BackupActions'
 import BackupDescription from './components/BackupDescription'
 import InstallAppAlert from './components/InstallAppAlert'
+import UpdateAppAlert from './components/UpdateAppAlert'
 import LastBackupStatus from './components/LastBackupStatus'
 import AllowPermissionsModal from './components/AllowPermissionsModal'
 import { BackupError } from './components/BackupError'
 
 import { BackupActionsProvider } from 'photos/ducks/backup/hooks/useBackupActions'
 import { isFlagshipApp } from 'cozy-device-helper'
 
-const BackupPage = () => {
+const BackupPageContent = () => {
```

Comment:
nit: Extract this component into a separate file

Reply:
This component will be used only here so I prefer to keep it here.